### PR TITLE
Add Affording food question

### PR DESCRIPTION
**Pull request by leenagupte**
Trello: https://trello.com/c/ayUNm2fN

Screenshot 2020-04-08 at 10 53 48
Screenshot 2020-04-08 at 10 53 54

### CODE REVIEW

File: spec/requests/afford_food.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
+RSpec.describe "afford-food" do
+  describe "GET /afford-food" do
+    let(:selected_option) { I18n.t("coronavirus_form.questions.afford_food.options").sample }
+
+    context "without session data" do
+      it "shows the form" do
+        visit are_you_off_work_ill_path
```
Comment (brucebolt) [inline spec/requests/afford_food.rb:7]:
```suggestion
        visit afford_food_path
```

Comment (leenagupte) [reply spec/requests/afford_food.rb:7]: Oops! Copy and Paste error 🤦‍♂

Comment (leenagupte) [reply spec/requests/afford_food.rb:7]: It's a bit worrying that the test passed. 🤔

Comment (leenagupte) [reply spec/requests/afford_food.rb:7]: Because I'd named the file incorrectly 🤦‍♂

**Comment by leenagupte** [comment]
@brucebolt I've updated and fixed up.

**Pull request merged**